process/accounts/accountsProcessor.go

```diff
@@ -220,38 +211,58 @@ func (ap *accountsProcessor) PrepareAccountsHistory(
 	return accountsMap
 }
 
-func (ap *accountsProcessor) getESDTInfo(accountESDT *data.AccountESDT) (*big.Int, string, error) {
+func (ap *accountsProcessor) getESDTInfo(accountESDT *data.AccountESDT) (*big.Int, string, *data.TokenMetaData, error) {
```

Comment:
could have made an unexported structure for passing these values (balance, properties and meta data) instead of 4 return values.

Reply:
I will let it as it is. no need for an extra tdo.